src/mcp_mt5/main.py

```
@@ -0,0 +1,1107 @@
+        flags: Type of requested ticks:
+            - mt5.COPY_TICKS_ALL: All ticks (default)
+            - mt5.COPY_TICKS_INFO: Ticks containing bid and/or ask price changes
+            - mt5.COPY_TICKS_TRADE: Ticks containing last price and volume changes
+        
+    Returns:
+        List[Dict[str, Any]]: List of ticks.
+    """
+    ticks = mt5.copy_ticks_from(symbol, start_pos, count, flags)
+    if ticks is None:
+        logger.error(f"Failed to copy ticks for {symbol}, error code: {mt5.last_error()}")
+        raise ValueError(f"Failed to copy ticks for {symbol}")
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)